modules/byo-vpc-multiple-az.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * assemblies/byo-vpc.adoc
+
+[id="byo-vpc-multiple-az_{context}"]
+= BYO VPC with multiple availability zones required fields
```

Comment:
This procedure module is missing the `.Procedure` heading and should also have an intro sentence. Some of the other modules are in the same boat - do you mind checking and updating others as well?